string action
---
uint8 result
